devai-cli/src/devai/commands/review.py

```diff
@@ -868,7 +868,7 @@ def image(file, prompt):
     
     contents = [qry, load_image_from_path(file)]
 
-    code_chat_model = GenerativeModel(MODEL_NAME)
+    code_chat_model = GenerativeModel(GEMINI_PRO_MODEL)
```

Comment:
lets keep it consistent with existing commands
https://github.com/GoogleCloudPlatform/genai-for-developers/blob/main/devai-cli/src/devai/commands/review.py#L297